src/commands/client.ts

```diff
@@ -350,18 +350,24 @@ export class CommandClient extends Client implements CommandClientOptions {
     }
 
     if (command.args !== undefined) {
-      if (typeof command.args === 'boolean' && parsed.args.length === 0)
-        return this.emit('commandMissingArgs', ctx)
-      else if (
-        typeof command.args === 'number' &&
-        parsed.args.length < command.args
-      )
-        this.emit('commandMissingArgs', ctx)
+      if (parsed.args.length === 0) {
```

Comment:
Maybe merged these `if` and `else if` since they execute same thing.

Reply:
Good catch, was able to merge it all into 1 if statement.